src/translations/fr.json

```
@@ -3,6 +3,7 @@
         "card": {
             "generic": {
                 "vertical": "Vertical ?",
+                "multiline_secondary": "Secondaire multiligne?",
```

```suggestion
                "multiline_secondary": "Information secondaire sur plusieurs lignes?",
```
For the french translation 🙂

---

Awesome 🚀I hope my google translation did sound too weird 😆